### PR TITLE
[#4519] Decrease horizontal space of form variable dropdowns

### LINKED ISSUE #4519: Form variable dropdowns take up too much horizontal space
### Product versie / Product version

2.7.0

### Customer reference

_No response_

### Omschrijf het probleem / Describe the bug

image

These dropdowns are sized based on the length of the options, which is excessively large with the new static variables present.

### Stappen om te reproduceren / Steps to reproduce

_No response_

### Verwacht gedrag / Expected behavior

_No response_

### Screen resolution

bigger

### Device

Desktop

### OS

Linux

### Browser

Chrome

### PR DESCRIPTION
**Pull request by vaszig**
Closes #4519

**Changes**

- Added class name to the `VariableSelection` in order to define `max-inline-size` for the form variable dropdowns

**Checklist**

Check off the items that are completed or not relevant.

- Impact on features

  - [x] Checked copying a form
  - [x] Checked import/export of a form
  - [x] Config checks in the configuration overview admin page
  - [x] Problem detection in the admin email digest is handled

- Release management

  - [x] I have labelled the PR as "needs-backport" accordingly

- I have updated the translations assets (you do NOT need to provide translations)

  - [x] Ran `./bin/makemessages_js.sh`
  - [x] Ran `./bin/compilemessages_js.sh`

- Commit hygiene

  - [x] Commit messages refer to the relevant Github issue
  - [x] Commit messages explain the "why" of change, not the how

### CODE REVIEW

File: src/openforms/scss/components/admin/_select.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+select.form-variable-dropdown {
+  max-inline-size: 300px !important;
+}
```
Comment (sergei-maertens) [inline src/openforms/scss/components/admin/_select.scss:3]: 1. CSS selectors should only target class names, not elements or IDs
2. why the `!important`? I was trying it out in storybook and don't see a difference with or without it

```suggestion
.form-variable-dropdown {
  max-inline-size: 300px;
}
```

Comment (vaszig) [reply src/openforms/scss/components/admin/_select.scss:3]: For me it wasn't working without the `!important`. I will update the css file, according to your suggestion,  and try it once more

Comment (sergei-maertens) [reply src/openforms/scss/components/admin/_select.scss:3]: I was looking at storybook - maybe there's something else in the admin that's interfering :thinking:

Comment (vaszig) [reply src/openforms/scss/components/admin/_select.scss:3]: Isn't the parent class that is present in the admin `dsl-editor__node`?As it appears in the screenshot?

**Review by sergei-maertens — APPROVED** [review]
will merge and backport tomorrow

**Comment by sergei-maertens** [comment]
Backports:

* stable/2.7.x: e575180fa70a3e439a679df69121856c8b5d85f6
* stable/2.6.x: 70da5e588396ce38d9638255e18528ad3dd680dc

2.5.x was not really affected by this problem

**Pull request merged**